pkg/apiserver/profile/handler.go

```diff
@@ -0,0 +1,248 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package profile
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pingcap/log"
+
+	"github.com/pingcap-incubator/tidb-dashboard/pkg/apiserver/httputil"
+)
+
+// Built-in component type
+const (
+	tikv = "tikv"
+	tidb = "tidb"
+	pd   = "pd"
+)
+
+// @Summary Create a task group
+// @Description create and run a task group
+// @Produce json
+// @Param tikv query string false "tikv"
+// @Param tidb query string false "tidb"
+// @Param pd query string false "pd"
+// @Success 200 {object} httputil.HTTPSuccess
+// @Router /profile/group/start [get]
+func (s *Service) startHandler(c *gin.Context) {
+	tg := NewTaskGroup()
+	addrs := make(map[string][]string)
+	addrs[tidb] = c.QueryArray(tidb)
```

Comment:
A better way is to:

1. `type StartRequest struct { ... }`
2. Use `c.ShouldBindJSON` to bind the request body to this handler.